libraries/DoublyLinkedList/DoublyLinkedList.h

```diff
@@ -0,0 +1,78 @@
+//  Created by Andrew Florjancic on 9/18/24.
+//  Copyright © 2024 Andrew Florjancic. All rights reserved
+#ifndef DoublyLinkedList_h
+#define DoublyLinkedList_h
+
+#include "DLLNode.h"
+
+// A Generic Doubly Linked List.
+template <class T> 
+class DoublyLinkedList {
+    private:
+        DLLNode<T>* head = nullptr; // A pointer to the first item in the list.
+        DLLNode<T>* tail = nullptr; // A pointer to the last item in the list.
+    public:
+        DLLNode<T>* current_item = nullptr; // A pointer to the current item in the list.
```

Comment:
This shouldn't be public. Make this private and add a get method to access the current item and not be able to modify it.

Reply:
Updated